.github/workflows/ci.yml

```diff
@@ -6,7 +6,7 @@ on:
       - 'packages/docs/**'
       - 'packages/playground/**'
   pull_request:
-    branches: v2
+    branches: [v2]
```

Comment:
Why?

Reply:
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore

This branches key is an Array. If defined in one line the Array structure should be provided.

Reply:
Are you sure? It seems to work just fine